src/types.ts

```diff
@@ -20,6 +22,7 @@ export type OnchainKitContextType = OnchainKitConfig;
 
 export type OnchainKitProviderProps = {
   address?: Address;
+  apiKey: string;
```

Comment:
should this be optional? Not all components or utilities will need an ApiKey, and some folks could use OnchainKit without the need of this.

Reply:
removed mandatory api key